Wikipedia/Code/WMFArticleFooterMenuDataSource.m

```diff
@@ -2,7 +2,7 @@
 #import "WMFArticleFooterMenuItem.h"
 #import "MWKArticle.h"
 #import "NSDate+Utilities.h"
-#import "WMFArticleFooterMenuCell.h"
+#import "WMFArticleListTableViewCell.h"
```

Comment:
Why are we removing the footer cell and replacing with a cell that is made for articles?

I don't think we want the layout of articles tied to the layout of footer items.